presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleNodeSelector.java

```diff
@@ -129,20 +130,46 @@ public SplitPlacementResult computeAssignments(Set<Split> splits, List<RemoteTas
         boolean splitWaitingForAnyNode = false;
 
         // todo identify if sorting will cause bottleneck
-        List<HostAddress> sortedCandidates = sortedNodes(nodeMap);
+        List<HostAddress> sortedCandidates = null;
```

Comment:
The `// todo` comment above is no longer relevant after this change and should be removed.